server-spi-private/src/main/java/org/keycloak/services/resource/AccountResourceSpi.java

```diff
@@ -0,0 +1,34 @@
+package org.keycloak.services.resource;
+
+import org.keycloak.provider.Provider;
+import org.keycloak.provider.ProviderFactory;
+import org.keycloak.provider.Spi;
+
+/**
+ * <p>A {@link Spi} to replace Account resources.
+ *
+ * <p>Implementors can use this {@link Spi} to override the behavior of the Account endpoints and resources by
+ * creating JAX-RS resources that override those served at /account by default.
+ */
+public class AccountResourceSpi implements Spi {
+
+    @Override
+    public boolean isInternal() {
+        return true;
+    }
+
+    @Override
+    public String getName() {
+        return "account-console-extension";
```

Comment:
I'd drop `extension` from the name, that's implied since it's an SPI.

```suggestion
        return "account-console";
```

Reply:
Updated